src/swarmus_pioneer/launch/pioneer_bringup.launch

```diff
@@ -33,14 +34,19 @@
             <param name="~max_angular_speed" value="1"/>
 
             <param name="~tf_prefix" value="$(arg robot_name)"/>
+            
+            <param name="~publish_tf" value="false" if="$(arg fuse_odom)"/>
```

Comment:
both expressions seems to do the same checks on the same variable, am I right?
false if (x)    is the same as:     true if not (x)

Reply:
Yeah at first is because I was not able to find a way to use a not opperation. But I just found out that the eval function let me use python opperation, so Im gonna fix it.